googleapiclient/discovery.py

```diff
@@ -343,14 +340,16 @@ def _discovery_service_uri_options(discoveryServiceUrl, version):
       A list of URIs to be tried for the Service Discovery, in order.
     """
 
-    urls = [discoveryServiceUrl, V2_DISCOVERY_URI]
-    # V1 Discovery won't work if the requested version is None
-    if discoveryServiceUrl == V1_DISCOVERY_URI and version is None:
+    if not discoveryServiceUrl is None:
```

Comment:
```suggestion
    if discoveryServiceUrl is not None:
```

Suggested change based on style guide: https://www.python.org/dev/peps/pep-0008/#programming-recommendations